polaris-react/src/components/Frame/Frame.scss

```diff
@@ -1,8 +1,13 @@
 @import '../../styles/common';
 
+$sidebar-breakpoint: 1200px;
+
 .Frame {
   // stylelint-disable-next-line -- Polaris component custom properties
   --pc-frame-button-size: var(--p-space-8);
+  --nav-width: 240px;
+  --topbar-height: 56px;
+  --sidebar-width: 376px;
```

Comment:
Not sure about these variables. Should they be somewhere else

Reply:
These are fine here, just suggested they have the Polaris component prefix.